src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRunTypeSystemHelper.cs

```diff
@@ -0,0 +1,386 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Internal.TypeSystem
+{
+    public static class ReadyToRunTypeSystemHelpers
```

Comment:
Nit: Class name doesn't match the filename

Reply:
Thanks for reminding me, I was telling myself a dozen times I should rename the file and I always forgot :-).